support/client/lib/vwf/view/kineticjs.js

```diff
@@ -1742,6 +1753,26 @@ define( [ "module", "vwf/view", "jquery", "vwf/utility", "vwf/utility/color" ],
         viewDriver.kernel.createChild( localDrawingInfo.drawingParentID,
             localDrawingInfo.drawingChildName, drawingDef );
 
+        var action = {
+            type: "drawing",
+            drawingInfo: localDrawingInfo,
+            drawingDef: drawingDef
+        };
+        action.Undo = function(){
```

Comment:
Nope. I can change them to lower case.  It was likely a rename. I think they were originally prepended with something, and I probably did a find/replace without thinking of adjusting for camelCase